File: rb/test/exists_test.rb
# RemoteOkJobs SDK exists test

require "minitest/autorun"
require_relative "../RemoteOkJobs_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = RemoteOkJobsSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- RemoteOkJobs SDK exists test

local sdk = require("remote-ok-jobs_sdk")

describe("RemoteOkJobsSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
